src/backend/joanie/core/serializers/client.py

```diff
@@ -240,17 +240,21 @@ class NestedOrderSerializer(serializers.ModelSerializer):
     id = serializers.CharField(read_only=True, required=False)
     course = CourseLightSerializer(read_only=True, exclude_abilities=True)
     organization = OrganizationSerializer(read_only=True, exclude_abilities=True)
-    product = serializers.SlugRelatedField(
-        queryset=models.Product.objects.all(), slug_field="title"
-    )
-    owner = serializers.SerializerMethodField()
+    product_title = serializers.SerializerMethodField()
+    owner_name = serializers.SerializerMethodField()
 
     class Meta:
         model = models.Order
-        fields = ["id", "course", "organization", "owner", "product"]
+        fields = ["id", "course", "organization", "owner_name", "product_title"]
         read_only_fields = fields
 
-    def get_owner(self, instance):
+    def get_product_title(self, instance):
+        """
+        Return the title of the product
+        """
+        return instance.product.title
```

Comment:
Should we add a 'prefetch_related' in list endpoints that use this OrderNestedSerializer ? (Certificate and Contract)

Reply:
Finally, I kept the existing serializer, no need to use a SerializerMethodField

Reply:
Sorry, my answer was silly.
select_related have been updated :+1: